tmail-backend/event-bus-redis/src/main/java/org/apache/james/events/TMailEventDispatcher.java

```diff
@@ -176,6 +182,11 @@ private Mono<Void> remoteDispatch(String eventAsJson, Collection<RoutingKey> rou
         return Flux.fromIterable(routingKeys)
             .flatMap(routingKey -> getTargetChannels(routingKey)
                 .flatMap(channel -> redisPublisher.publish(channel, KeyChannelMessage.from(eventBusId, routingKey, eventAsJson).serialize()))
+                .timeout(redisEventBusConfiguration.durationTimeout())
+                .onErrorResume(REDIS_ERROR_PREDICATE.and(e -> redisEventBusConfiguration.failureIgnore()), e -> {
+                    LOGGER.warn("Error while dispatching event to remote listeners: {}", e.getMessage());
```

Comment:
LOGGER.warn("Error while dispatching event to remote listeners", e);

Otherwise precious stack trace is lost!